--- build/pdfium/.gn	2017-07-11 13:15:34.949665680 +0100
+++ build/pdfium/new.gn	2017-07-11 13:15:24.453631341 +0100
@@ -5,10 +5,11 @@
 buildconfig = "//build/config/BUILDCONFIG.gn"
 secondary_source = "//build/secondary/"
 
+# comment out ... these break the build if v8 is disabled
 default_args = {
-  v8_extra_library_files = []
-  v8_experimental_extra_library_files = []
+  # v8_extra_library_files = []
+  # v8_experimental_extra_library_files = []
 
   # Turns on compiler optimizations in V8 in Debug build.
-  v8_optimized_debug = true
+  # v8_optimized_debug = true
 }
